### ISSUE TITLE
Learning Texture Invariant Representation for domain Adaptation of semantic Segmentation

### ISSUE THREAD

**Issue by karmenkokoko** [karmenkokoko/Papermining#1]
#What is the topic?
Train model on synthetic data set to learn the target texture

#Why this topic?
manual annotation need time
to learn texture-invariant representation

#What is the challenge?
generate data may not be ideal in diverse texture
labeled training data is scarce
model may overfitting on one specific texture

#What is the keyt-diversified datasets with style transfer.
2. train model on diver-data-sets to learn texture invariant representation
3. using self training to finetune the model to target domain. using pseudo-labels on target training images.
